### PR TITLE
Display all Expense category

### PR DESCRIPTION
**Pull request by timtech-wiz**

### CODE REVIEW

**Comment by timtech-wiz** [comment]
Screenshot (13)

File: login.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+              <div class="px-8 py-6">
+                <label for="" class="block font-semibold">Email:</label>  
+              <input type="email" placeholder="Enter Email" name="email" class="border-w-full h-5 px-3 py-5 md-2 hover:outline:none focus-ring-1 focus:ring-indigo-400 rounded-md">
+             
+            <div class="flex justify-center items-baseline">
+                <button type="submit" class="mt-4 bg-indigo-500 text-white py-2 px-6 rounded-md hover:bg-indigo-600">Login</button>
+            </div>
+            </div>
+        </form>
+
```
Comment (holuborhee) [inline login.html]: what's the use of these codes you commented out? Looks like the whole file is not even needed why do we have it here?

File: src/App.vue [hunk 2]
```diff
@@ -1,26 +1,28 @@
...
-          </div>
-        </div>
-      </nav>
- 
-    
- <router-view></router-view>
+  <div class="flex flex-col h-screen justify-between">
+      
+
+      <the-header></the-header>
```
Comment (holuborhee) [inline src/App.vue]: This is meant to be written as `<TheHeader/>`. Kindly refer to the following links

https://v3.vuejs.org/style-guide/#self-closing-components-strongly-recommended
https://v3.vuejs.org/style-guide/#component-name-casing-in-templates-strongly-recommended
```suggestion
      <TheHeader/>
```

Correct all other such instances where you repeated the same approach

**Review by holuborhee — CHANGES_REQUESTED** [review]
Kindly note the requested changes. 

Also, check out this section of the doc https://v3.vuejs.org/style-guide/#order-of-words-in-component-names-strongly-recommended. Read the detailed explanation for an understanding of the recommendation when it comes to putting components in folders. I am still not comfortable with your folder structure. You should have two folders `components` and `views`. Pages should be inside the views folder, components that aren't pages should be in the components folder and there should be no subdirectory.

**Pull request merged**